files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js

```diff
@@ -3216,22 +3227,26 @@ MyApplet.prototype = {
         }
     },
 
+    _matchNames: function(names, pattern){
+        var res = [];
```

Comment:
Although older code is using `var`, in new code we use lexical scoping with `let`.